cleanup celo toml

ericnakagawa opened the pull request:
Added new community gh organizations, removed defunct projects

bcamacho commented:
@jubos can you please take a look at this PR to review/approve @ericnakagawa workflow for merge?

ericnakagawa commented:
added an additional batch of accelerator projects @jubos any feedback is appreciated

jubos commented on data/ecosystems/c/celo.toml:
Hyperlane should be its own ecosystem. Added here: [e817889](https://github.com/electric-capital/crypto-ecosystems/commit/e8178891ae6d7c171a9dd0c7d16ac0870f7c2259)

jubos replied:
Hyperlane should just be its own top level ecosystem outside of Celo.  If there is a specific repo from Hyperlane in the Celo ecosystem, we could add it in the celo.toml file.

jubos commented on data/ecosystems/c/celo.toml:
```diff
@@ -25,10 +25,112 @@ github_organizations = [
   "https://github.com/ReSource-Network",
   "https://github.com/Tradegen",
-  "https://github.com/centfinance"
+  "https://github.com/centfinance",
+  "https://github.com/NEFTME/",
+  "https://github.com/hyperlane-xyz/",
+  "https://github.com/celo-tracker",
```
hyperlane is a multi ecosystem, so isn't limited to just celo.

jubos commented on data/ecosystems/c/celo.toml:
```diff
@@ -25,10 +25,112 @@ github_organizations = [
   "https://github.com/ReSource-Network",
   "https://github.com/Tradegen",
-  "https://github.com/centfinance"
+  "https://github.com/centfinance",
```
cent.finance is also multi ecosystem.  A lot of multi network wallts wallets can either be sub ecosystems of General or just exist by themselves since it is hard to attribute wallet work to any one particular ecosystem.

ericnakagawa replied:
how should we best attribute these be multichain type projects?

bcamacho replied:
@jubos We invest in working with more comprehensive ecosystem partners that are multi-chain focus. How can we ensure contributions to these efforts are reflected in metrics for the end-of-year developer report with Celo?

ericnakagawa replied:
I think a "bridged" or "multichain" type is needed here in future, but with the option for identifying which chain is "home".

jubos commented on data/ecosystems/c/celo.toml:
```diff
@@ -893,7 +995,7 @@ url = "https://github.com/celo-org/build-with-celo-hackathon"
 [[repo]]
 url = "https://github.com/celo-org/celo-blockchain"
-tags = ["Protocol"]
+tags = [ "Protocol" ]
```
🔥

ericnakagawa replied:
Used a new tool to convert from TOML to JSON and back.

jubos commented on data/ecosystems/c/celo.toml:
```diff
@@ -1020,7 +1122,7 @@ url = "https://github.com/celo-org/celoterminal"
 [[repo]]
 url = "https://github.com/celo-org/CIPs"
-tags = ["RFC"]
+tags = [ "RFC" ]
```
🔥

jubos reviewed (changes requested):
Thank you for these repos.  There are a few tweaks and then I think it is ready to go.

ericnakagawa commented on data/ecosystems/c/celo.toml:
pinnata is a tiny project, what is the threshold before spinning out? also how to attribute it back to Celo ecosystem.

bcamacho commented on data/ecosystems/c/celo.toml:
```diff
@@ -21,9 +20,22 @@ github_organizations = [
+  "https://github.com/Ubeswap",
+  "https://github.com/ReSource-Network",
+  "https://github.com/Tradegen",
+  "https://github.com/centfinance"
```
@jubos +1 to @ericnakagawa request for best practice document regarding the usage of ecosystem spinouts. We want to follow best practices here but also be mindful that once spun out, we can maintain active contribution efforts to our ecosystem as a whole.

jubos replied:
You can just make them sub ecosystems of Celo if they are direct spinouts.  A good rule of thumb is if a project has its own discord/twitter/github org then it probably is its own ecosystem.

ericnakagawa commented:
@jubos can you review the latest changes here?

ericnakagawa commented:
> Looks like it is getting closer. Can you fix the build error? It looks like it is missing Node-Fi and ReSource Network sub ecosystems.

I've added both, and cleaned up the celo.toml file. PTAL

ericnakagawa commented:
@jubos ready for review!

ericnakagawa commented:
@jubos I can't trigger any build issues, PTAL.

ericnakagawa commented:
@jubos Issues appear cleared from my end.

jubos commented:
Looks great, thank you for working through all the comments!
